pkg/workflow_execution.go

```diff
@@ -466,7 +469,16 @@ func (c *Client) injectAccessForSidecars(namespace string, wf *wfv1.Workflow) ([
 		for si, s := range t.Sidecars {
 			//If TTY is true, sidecar needs to be accessible by HTTP
 			//Otherwise, we skip the sidecar
-			if s.TTY != true {
+
+			hasInjectIstio := false
+			for _, e := range s.Env {
+				if e.Name == "inject-istio" && e.Value == "true" {
```

Comment:
Ideally we should have the env var indicate that this is an interactive sidecar... something like `ONEPANEL_INTERACTIVE_SIDECAR=true`

Reply:
@rushtehrani Good idea - on it.